src/commercial.ts

```diff
@@ -0,0 +1,39 @@
+import {
+	getConsentFor,
+	onConsent,
+} from '@guardian/consent-management-platform';
+import { bootCommercialWhenReady } from 'init/consented';
+import { commercialFeatures } from 'lib/commercial-features';
+
+/**
+ * Choose whether to launch Googletag or Opt Out tag (ootag) based on consent state
+ */
+const chooseAdvertisingTag = async () => {
+	const consentState = await onConsent();
+	// Only load the Opt Out tag if:
+	// - in TCF region
+	// - no consent for Googletag
+	// - the user is not a subscriber
+	if (
+		consentState.tcfv2 &&
+		!getConsentFor('googletag', consentState) &&
+		!commercialFeatures.adFree
+	) {
+		void import(
+			/* webpackChunkName: "consentless" */
+			'./init/consentless'
+		).then(({ bootConsentless }) => bootConsentless(consentState));
+	} else {
+		bootCommercialWhenReady();
```

Comment:
Would it be worth code splitting the consented init, so we don't have to load it if there's no consent?

Reply:
Looks like we decided to split it off for the AB test and never revisited https://github.com/guardian/frontend/pull/25682

Reply:
Ah cool, good detectiving! Maybe we can investigate whether this would be worth it as a separate ticket? Wouldn't want to delay merging this change

Reply:
I think it's worth thinking a bit about this seperatly, I'll create a card to follow up